src/docker/run-apache.sh

```diff
@@ -2,9 +2,18 @@
 set -euo pipefail
 IFS=$'\n\t'
 
-cp /config/config.json /app/target/config.json
 
-SITE_CONF=$(cat << EOF
+CONFIG_JSON=$(cat <<EOF
```

Comment:
why do it this way? This seems to go against the model we're trying to set, where config files are stored in firecloud-develop. (or firecloud)